infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -367,14 +314,7 @@ public synchronized void trimCache(CorfuPayloadMsg<Long> msg,
             trimMark = msg.getPayload();
         }
 
-        long entries = 0;
-        for (Map.Entry<String, Long> entry : conflictToGlobalTailCache.asMap().entrySet()) {
-            if (entry.getValue() < trimMark) {
-                conflictToGlobalTailCache.invalidate(entry.getKey());
-                entries++;
-            }
-        }
-        log.info("trimCache: Evicted {} entries", entries);
+        cache.invalidateUpTo(trimMark);
```

Comment:
If the trim mark doesn't change why do you still iterate the cache ?

Reply:
fixed